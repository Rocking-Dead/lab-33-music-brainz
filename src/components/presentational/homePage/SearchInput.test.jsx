import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';
import SearchBar from './SearchInput';


describe('SearchBar component', () => {
  afterEach(() => cleanup());
  it('renders SearchBar', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <SearchBar search={''} handleSearch={jest.fn()} />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
