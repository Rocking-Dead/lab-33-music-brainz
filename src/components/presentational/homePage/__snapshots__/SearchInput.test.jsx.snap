// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component renders SearchBar 1`] = `
<DocumentFragment>
  <form>
    <input
      value=""
    />
    <button>
      Search
    </button>
  </form>
</DocumentFragment>
`;
